tests/models/whisper/test_modeling_whisper.py

```diff
@@ -2520,58 +2526,19 @@ def test_default_multilingual_transcription_long_form(self):
         # task defaults to transcribe
         sequences = model.generate(input_features, return_timestamps=True)
 
-        transcription = processor.batch_decode(sequences)[0]
+        transcription = processor.batch_decode(sequences, skip_special_tokens=False)[0]
 
         assert transcription == " मिर्ची में कितने विबिन्द प्रजातियां हैं? मिर्ची में कितने विबिन्द प्रजातियां हैं?"
 
         # set task to translate
         sequences = model.generate(input_features, task="translate", return_timestamps=True)
-        transcription = processor.batch_decode(sequences)[0]
+        transcription = processor.batch_decode(sequences, skip_special_tokens=False)[0]
 
         assert (
             transcription
             == " How many different species are there in the chilli? How many different species are there in the chilli?"
         )
 
-    @slow
-    def test_generate_with_prompt_ids_and_forced_decoder_ids(self):
```

Comment:
Same comment than before: this test and the following one are checking that prompt ids and decoder ids are still in the generated output. Should we instead check that the overall generated tokens are aligned with the ones we'd have with the original implementation ?

Reply:
Indeed, like said above, we have tests that verify that the output is aligned when setting `prompt_ids=`: `test_whisper_longform_prompt_ids`, `test_generate_with_prompt_ids`. Yet, we do (and did) not have any when setting `forced_decoder_ids` (the one we had was just checking that the generated sequence had the forced ids in the beginning). Let me add one!

Reply:
test added in commit [e5064d9](https://github.com/huggingface/transformers/pull/34111/commits/e5064d9b74ce5847075d52245d8414dea427dffe)